tests/UnifiedSpecTests/Operation.php

```diff
@@ -471,18 +471,12 @@ private function executeForBucket(Bucket $bucket)
             case 'delete':
                 return $bucket->delete($args['id']);
             case 'downloadByName':
-                return stream_get_contents($bucket->openDownloadStream(
+                return stream_get_contents($bucket->openDownloadStreamByName(
                     $args['filename'],
                     array_diff_key($args, ['filename' => 1])
                 ));
             case 'download':
                 return stream_get_contents($bucket->openDownloadStream($args['id']));
-            case 'uploadWithId':
-                $args['_id'] = $args['id'];
-                unset($args['id']);
-
-                // Fall through
-
```

Comment:
The unified test format spec still references this method, can you create a ticket or PR to https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#upload-and-uploadwithid?

Reply:
Oh, I totally missed that. Will revert this change.

The method should absolutely exist, as it's noted in the [GridFS spec](https://github.com/mongodb/specifications/blob/master/source/gridfs/gridfs-spec.rst#file-upload) itself.